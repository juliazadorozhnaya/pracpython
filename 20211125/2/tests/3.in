Êîðåöêèé Àðòåìèé Êèëîòîí Çà÷åì ïîáåæàë
