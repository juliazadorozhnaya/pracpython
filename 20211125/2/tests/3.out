Корецкий Артемий Килотон Зачем побежал
